src/app/components/explore-code/repo/repo.component.html

```diff
@@ -0,0 +1,84 @@
+<div class="repo-general">
+
+    <!-- <div class="repo-banner">
+      <h1>{{repo.agency.acronym}}</h1>
+      <h2>{{repo.name}}</h2>
+      
+      <a *ngIf="getRepositoryUrl() | isdefined" href="{{ getRepositoryUrl() }}">
+          <button class="button button--transparent">Visit Repo</button>
+        </a>
+    </div> -->
+  
+
+
+<section *ngIf="repo" class="repo-container usa-grid">
+    <div class="repo-breadcrumb">
+        <ul class="usa-unstyled-list">
+          <li><a routerLink="/">Home</a></li>
+          <li><a routerLink="/explore-code/agencies/{{repo.agency.acronym}}">{{repo.agency.acronym}}</a></li>
+          <li>{{repo.name}}</li>
+        </ul>
+      </div>
+    <header>
+      <div class="repo-header-container">
+        <h2>{{ repo.name }}</h2>
+        <span *ngIf="repo.date?.metadataLastUpdated" class="updated">
+          <div>Last updated on {{repo.date.metadataLastUpdated | date: "MM/dd/yy"}}</div>
+        </span>
+        <span *ngFor="let tag of repo.tags">
+          <button *ngIf="tag" 
```

Comment:
totally, not a big deal and I'll still accept merge without it, but it would be cool if somehow you can use an a wrapped around a button and attach routerLink to that. That way, a user can see where they'll be headed if they hover over.  Totally not a big deal though. For future reference mostly :-)